### PR TITLE
New build scripts

### PR DESCRIPTION
**Pull request by mehmetkillioglu**
The new build scripts added for standalone package building without fogsw-ci-scripts. 

It can be build for foxy and galactic. The build strategy is added to run two tasks and upload the foxy and galactic built deb packages to JFrog.

### CODE REVIEW

**Comment by mehmetkillioglu** [comment]
Seems like it is also running the .github/workflows/build.yml, but there is no need for that. It is enough to run .github/workflows/main.yaml only.

**Comment by mehmetkillioglu** [comment]
The motivation of running builds for foxy and galactic is to ease up galactic porting work. Similar scripts are going to be created other ROS2 nodes and the checks can be done for Galactic also.

The artifactory does not have any galactic deb packages yet. Since the build for px4-msgs takes to longest time, it would be good to have it on artifactory at the beginning.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    dh-make \
+    libboost-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN groupadd -g $GID builder && \
+    useradd -m -u $UID -g $GID -g builder builder && \
+    usermod -aG sudo builder && \
+    echo 'builder ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+RUN echo "deb [trusted=yes] https://ssrc.jfrog.io/artifactory/ssrc-debian-public-remote focal fog-sw" >> /etc/apt/sources.list
```
Comment (kulmesa) [inline Dockerfile:25]: why is this needed here?

Comment (mehmetkillioglu) [reply Dockerfile:25]: Not needed for this package. We created some template scripts that can be used for other packages as well. For instance, the fog_bumper would have fog-msgs dependency, so it can get from the artifactory repo. 

The scripts also have ability to fetch dependencies using underlay.repos if the requested version is not available in the artifactory using vcs tool.

Comment (mehmetkillioglu) [reply Dockerfile:25]: The example for this dependency gathering can be found in [odometry2/DP-1761-new_build_process](https://github.com/tiiuae/odometry2/tree/DP-1761-new_build_process). 

If the requested version does not exist in the artifactory, it will get the package using underlay.repos, build it and create the deb.

**Review by jlaitine — APPROVED** [review]

**Comment by mehmetkillioglu** [comment]
At the moment, the build scripts are not optimized for each node. It has unnecessary components, such as rosdep and vcs tool dependency gathering for this px4-msgs, since it does not have any special dependencies. However, it would be good to start having the galactic versions on JFrog for later development. The build scripts can be optimized later.
